src/core/core.datasetController.js

```diff
@@ -277,6 +277,7 @@ helpers.extend(DatasetController.prototype, {
 		var me = this;
 		var type = me.dataElementType;
 		return type && new type({
+			_animation: me.chart._animation ? true : false,
```

Comment:
If you passed this into `pivot` and `transition` as a parameter it would save you from having to create 150,000+ instances of the variable